influxql/parser.go

```diff
@@ -1508,6 +1508,8 @@ func (p *Parser) parseUnaryExpr() (Expr, error) {
 	case DURATION_VAL:
 		v, _ := ParseDuration(lit)
 		return &DurationLiteral{Val: v}, nil
+	case MUL:
+		return &Wildcard{}, nil
```

Comment:
Wondering if wildcard expressions should be parsed separately so we don't allow weird stuff like `SELECT value + *,* FROM cpu`, which will get expanded to look something like `SELECT value + *, value1, value2 FROM cpu`.  /cc @benbjohnson